src/components/Background.js

```diff
@@ -1,11 +1,43 @@
 import React from "react";
 import Header from './header/Header';
+import styles from './Background.module.css';
+import InfoBlock from "./info_block/InfoBlock";
 
 export default function Background({ data }){
+  const { label, education, experience } = data.page_spec_data;
+  console.log(education);
+  let testing = [...education];
+
+  const education_list = [];
+  for(let i = 0; i < 3; i++){
+    education_list[i] = education[Object.keys(education)[(1-i)]];
+  }
+  const experience_list = [];
+  for(let i = 0; i < 3; i++){
+    experience_list[i] = experience[Object.keys(experience)[(1-i)]];
+  }
+
   return (
-    <div className="Backpage">
+    <div className="background">
```

Comment:
Maybe change to fragments since, this does not modify anything.

Reply:
Decided that it's clearer to leave these in. Makes it easier to read, especially in browser check mode.